src/layers/core/scatterplot-layer/scatterplot-layer.js

```diff
@@ -109,6 +154,17 @@ export default class ScatterplotLayer extends Layer {
     }
   }
 
+  calculateInstancePositions64xyLow(attribute) {
+    const {data, getPosition} = this.props;
+    const {value} = attribute;
+    let i = 0;
+    for (const point of data) {
+      const position = getPosition(point);
+      value[i++] = fp64ify(position[0])[1];
```

Comment:
I think I added an `fp64lowPart`. Not sure if I merged yet. Maybe clearer than indexing this result?